### DISCUSSION TITLE
How to enable other nordvpn settings

### DISCUSSION THREAD

**Discussion by ic3blue** [tmknight/docker-nordvpn#105] [General]
Hello

First time to starting the discussion: can someone please help me on how to enable nordvpn other settings for this docker. The settings I have in mind are:

Analytics change to disable
Threat protection lite change to enable
Lan discovery change to enable 
post quantum change to enable

What I had done was: 
- login to nordvpn docker manually: sudo docker exec -ti (container name) /bin/bash
- Then manually changed the settings as above
- However, every time I need to reboot my linux box, the setting reverted back to original
- I already put the extra settings that I required to NordVPN.env, but it seems that is not working

My nordvpn.env settings:
TOKEN=secret
NET_LOCAL=192.168.1.0/24
FIREWALL=TRUE
TZ=local zone
REFRESH_CONNECTION_INTERVAL=1440
LAN_DISCOVERY=TRUE
pq=TRUE
ANALYTICS=FALSE
THREAT_PROTECTION_LITE=TRUE

**Comment by tmknight** [comment]
These have not been incorporated into the project - I have no immediate plans to do so.  Feel free to fork, modify and submit a PR to have the changes integrated.

**Comment by ic3blue** [comment]
Thank you … unfortunately I do not know how to do it 😊
